css/mark-novicio.css

```diff
@@ -0,0 +1,26 @@
+* {
+    font-family: 'Raleway', sans-serif;
+    line-height: 200%;
+}
+
+h1 {
+    color: #ff0000;
+}
+
+#first-list {
+    color: #00ff00
+}
+
+.date {
```

Comment:
Is there a way to apply these styles using an inline element, rather than a block level `<div>`?